Makefile

```diff
@@ -50,6 +50,18 @@ ci_bench_links: sys_links
 	docker run -v $(shell pwd):/source $(DOCKERHUB):links \
 		make -C /source/benchmarks/links
 
+# libhandler
+sys_libhandler:
+	docker build -t $(DOCKERHUB):libhandler systems/libhandler
+
+bench_libhandler: sys_libhandler
+	docker run -it --init -v $(shell pwd):/source $(DOCKERHUB):libhandler \
+		make -C /source/benchmarks/libhandler
```

Comment:
Can you add a (possibly empty)?
This will fix CI build.

Reply:
Done!